lib/std/crypto/25519/ed25519.zig

```diff
@@ -506,7 +507,7 @@ test "signature" {
 test "batch verification" {
     var i: usize = 0;
     while (i < 100) : (i += 1) {
-        const key_pair = try Ed25519.KeyPair.create(null);
+        const key_pair = try Ed25519.KeyPair.initWithRandomSeed();
```

Comment:
unit tests should not use `std.crypto.random`, instead they should use `std.testing.random_seed`.

this is a problem with master branch, but it should be fixed in this PR before merging.

Reply:
I noticed https://github.com/ziglang/zig/blob/master/lib/std/crypto/salsa20.zig#L539 will need to be changed to take a seed, the `initWithRandomSeed` should probably be removed to reduce the likelihood of similar mistakes.